src/example/FileInputForm.js

```diff
@@ -77,12 +97,31 @@ const FileInputForm = ({ form }) => (
 				/>
 			</label>
 		</div>
+		<div style={{ marginBottom: 10 }}>
+			<label htmlFor="cv">Select multiple files</label>
```

Comment:
Should probably be `htmlFor="attachments"` yeah?

Reply:
Um, yep.